versions.tf

```
@@ -1,3 +1,13 @@
 terraform {
   required_version = ">= 0.12.14"
```

```suggestion
  required_version = ">= 0.12.19"
```

just to be consistent with what we've been enforcing in general.